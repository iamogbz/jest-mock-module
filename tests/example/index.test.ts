import helloWorld from "src/example";

describe("example", (): void => {
    it("runs a test", (): void => {
        expect(helloWorld()).toMatchSnapshot();
    });
});
