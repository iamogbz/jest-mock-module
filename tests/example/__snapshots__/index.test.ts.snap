// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example runs a test 1`] = `"Hello world"`;
